src/udiff/_uarray_plug.py

```diff
@@ -109,33 +84,20 @@ def __ua_convert__(value, dispatch_type, coerce):
             with udiff.SKIP_SELF:
                 return DiffArray(np.asarray(value))
 
-        return NotImplemented
```

Comment:
Why this change?

Reply:
Because in the NoGradBackend, we need to return the raw value of `np.array`.

Reply:
What problems do you run into without removing this line? If not removed, it would return `None` which isn't an array anyway. In which case we should raise an error.

Reply:
When we set NoGradBackend, the `coerce` is `False` and the `value` is a `np.array`. Then we need to return the raw value, not a `DiffArray`. 
The code in the `backward` will go wrong without removing this line.

Reply:
If we return `NotImplemented`, it will go to the next backend anyway, which will be a `np.ndarray`.

Reply:
You can test it by the code:
```python
import uarray as ua
import unumpy as np
import unumpy.numpy_backend as NumpyBackend
import udiff

if __name__ == "__main__":
    with ua.set_backend(NumpyBackend), ua.set_backend(udiff, coerce=True):
        x1 = np.array([[2, 2], [2, 2]])
        x2 = np.array([[5, 5], [5, 5]])
        y1 = np.log(x1) + x1 * x2 - np.sin(x2)
        print(y1)
        y1.backward()
        print(x1.diff)
        print(x2.diff)
```